modules/porous_flow/src/materials/PorousFlowVolumetricStrain.C

```diff
@@ -35,6 +35,7 @@ PorousFlowVolumetricStrain::PorousFlowVolumetricStrain(const InputParameters & p
     _vol_strain_rate_qp(declareProperty<Real>("PorousFlow_volumetric_strain_rate_qp")),
     _dvol_strain_rate_qp_dvar(declareProperty<std::vector<RealGradient> >("dPorousFlow_volumetric_strain_rate_qp_dvar")),
     _vol_total_strain_qp(declareProperty<Real>("PorousFlow_total_volumetric_strain_qp")),
+    _vol_total_strain_qp_old(declarePropertyOld<Real>("PorousFlow_total_volumetric_strain_qp")),
```

Comment:
In a Material with a simple stateful initialization you no longer need to declare it as "old" or "older", or bother to initialize it. MOOSE will automatically promote that material to stateful if a kernel, or some other object requests the old state.